src/frame/endpoints/UserApiEndpoints.ts

```diff
@@ -179,6 +179,21 @@ const userUpdate = (userID: string, userPrivateKey?: PrivateKey<Uint8Array>, sta
     errorCode: ErrorCodes.USER_UPDATE_REQUEST_FAILURE,
 });
 
+const userKeyUpdateApi = (userID: string, userPrivateKey: PrivateKey<Uint8Array>, augmentationFactor: AugmentationFactor, userKeyId: number): RequestMeta => ({
+    url: `users/${encodeURIComponent(userID)}/keys/${userKeyId}`,
+    options: {
+        method: "PUT",
+        headers: {
+            "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+            userPrivateKey,
+            augmentationFactor,
```

Comment:
You need to convert these byte arrays into base64 before you JSON stringify them. The `fromByteArray` method that is already imported in this file is what you'll want to use.